includes/class-newspack-newsletters-settings.php

```diff
@@ -56,7 +56,7 @@ public static function get_settings_list() {
 			),
 			array(
 				'description' => __( 'Mailchimp API Key', 'newspack-newsletters' ),
-				'key'         => 'newspack_newsletters_mailchimp_api_key',
+				'key'         => 'newspack_mailchimp_api_key',
```

Comment:
This is a little bit problematic, because a user will see empty value here when they do have the legacy option set, but not the new one. The `default` key could be set here to mitigate this:

```
'default'     => get_option( 'newspack_newsletters_mailchimp_api_key', '' ),
```

Reply:
Ah, you're right, I forgot the default value here 👍